### PR TITLE
Remove invalid debuginfo symlink

### PR DESCRIPTION
**Pull request by sxa**
Fixes https://github.com/adoptium/adoptium-support/issues/298 (Rogue symlink to something that gets deleted)

Signed-off-by: Stewart X Addison <sxa@redhat.com>

### CODE REVIEW

**Review by karianna — APPROVED** [review]
LGTM

**Review by andrew-m-leonard — APPROVED** [review]
good

**Review by M-Davies — APPROVED** [review]
LGTM

**Comment by sxa** [comment]
(Note to committers: While this change should be safe, I haven't checked that this definitely works ... Will aim to do that tomorrow so I recommend holding off merging for now then we can verify before the change his the nightlies)

**Review by gdams — APPROVED** [review]

**Comment by sxa** [comment]
This looks to be good - merging

**Comment by breun** [comment]
I think that instead of `| xargs rm -f` you could also use the `-delete` flag of the `find` command?

```
     -delete
             Delete found files and/or directories.  Always returns true.  This executes from the current working directory as find recurses down the tree.  It will not attempt to
             delete a filename with a ``/'' character in its pathname relative to ``.'' for security reasons.  Depth-first traversal processing is implied by this option.  Following
             symlinks is incompatible with this option.
```

**Comment by sxa** [comment]
> I think that instead of `| xargs rm -f` you could also use the `-delete` flag of the `find` command?

It's not a standard option across all UNIX-based platforms.

**Pull request merged**
